internal/handlers/pg/msg_getlog.go

```diff
@@ -93,12 +93,19 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 
 		// TODO https://github.com/FerretDB/FerretDB/issues/1443
```

Comment:
I see the review is requested, but the state of the PR is draft. I guess it's not ready yet :)

Reply:
Yeah, it was ready for review but I've started thinking about the definition of done (how should I modify tests and what with metrics), though I can see that you've spotted that I forgot removing TODOs, so thanks!

Reply:
As agreed on the meeting - let's skip tests subject for now, unless someone has some easy solution to make them